internal/services/oracle/oracle_autonomous_database_clone_data_source.go

```diff
@@ -0,0 +1,366 @@
+			}
+
+			model.Name = id.AutonomousDatabaseName
+			model.ResourceGroupName = id.ResourceGroupName
+
+			if m := resp.Model; m != nil {
+				model.Location = m.Location
+				model.Tags = pointer.From(m.Tags)
+
+				// Verify this is actually a clone database
+				props, ok := m.Properties.(autonomousdatabases.AutonomousDatabaseCloneProperties)
+				if !ok {
+					return fmt.Errorf("%s is not a clone type autonomous database", id)
+				}
+
+				// Clone-specific properties
+				model.CloneType = string(props.CloneType)
+				model.SourceId = props.SourceId
+				model.IsReconnectCloneEnabled = pointer.From(props.IsReconnectCloneEnabled)
+				model.IsRefreshableClone = pointer.From(props.IsRefreshableClone)
+				model.TimeUntilReconnectClone = pointer.From(props.TimeUntilReconnectCloneEnabled)
+
+				if props.RefreshableModel != nil {
+					model.RefreshableModel = string(*props.RefreshableModel)
+				}
+				if props.RefreshableStatus != nil {
+					model.RefreshableStatus = string(*props.RefreshableStatus)
+				}
+
+				// Base properties
+				model.AutonomousDatabaseId = pointer.From(props.AutonomousDatabaseId)
+				model.BackupRetentionPeriodInDays = pointer.From(props.BackupRetentionPeriodInDays)
+				model.CharacterSet = pointer.From(props.CharacterSet)
+				model.ComputeCount = pointer.From(props.ComputeCount)
+				model.ComputeModel = string(pointer.From(props.ComputeModel))
+				model.CustomerContacts = flattenAdbsCustomerContacts(props.CustomerContacts)
+				model.DataStorageSizeInGbs = pointer.From(props.DataStorageSizeInGbs)
+				model.DataStorageSizeInTbs = pointer.From(props.DataStorageSizeInTbs)
+				model.DbVersion = pointer.From(props.DbVersion)
+				model.DbWorkload = string(pointer.From(props.DbWorkload))
+				model.DisplayName = pointer.From(props.DisplayName)
+				model.LicenseModel = string(pointer.From(props.LicenseModel))
+				model.AutoScalingEnabled = pointer.From(props.IsAutoScalingEnabled)
+				model.AutoScalingForStorageEnabled = pointer.From(props.IsAutoScalingForStorageEnabled)
+				model.MtlsConnectionRequired = pointer.From(props.IsMtlsConnectionRequired)
+				model.NationalCharacterSet = pointer.From(props.NcharacterSet)
+				model.SubnetId = pointer.From(props.SubnetId)
+				model.VnetId = pointer.From(props.VnetId)
+				model.PrivateEndpoint = pointer.From(props.PrivateEndpoint)
+				model.PrivateEndpointIp = pointer.From(props.PrivateEndpointIP)
+				model.ServiceConsoleUrl = pointer.From(props.ServiceConsoleURL)
+				model.SqlWebDeveloperUrl = pointer.From(props.SqlWebDeveloperURL)
+				model.TimeCreated = pointer.From(props.TimeCreated)
+				model.OciUrl = pointer.From(props.OciURL)
+
+				if props.LifecycleState != nil {
+					model.LifecycleState = string(*props.LifecycleState)
+				}
+
+				// Extract connection strings if available
+				if props.ConnectionStrings != nil && props.ConnectionStrings.AllConnectionStrings != nil {
+					connStrings := []string{}
```

Comment:
Nit: we generally initialize slices as:

```suggestion
					connStrings := make([]string, 0)
```